service/service

```diff
@@ -142,6 +144,10 @@ _d_upgrade () {
 
 	# run any database migrations/updates needed
 	sudo -u $PHAB_USER $ROOT/phabricator/bin/storage upgrade --force
+
+    # clear out old backups
+    cd $BAKPATH
+    ls -tp | grep -v revlog | tail -n +$(( $MAX_BACKUPS + 1)) | xargs -d '\n' rm -rf --
```

Comment:
This is starting to look a little too bash-y for me~
I'm guessing this:
1. Lists all files in the backup directory, ordered by date (descending?)
2. Filters out specific files with grep?
3. Lists only the last N+1 items
4. Does `rm -rf` with those last items